src/Rector/Deprecation/ConstantToClassConstantRector.php

```diff
@@ -2,9 +2,12 @@
 
 namespace DrupalRector\Rector\Deprecation;
 
+use DrupalRector\Rector\ValueObject\ConstantToClass;
 use PhpParser\Node;
 use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
 use Rector\Core\Rector\AbstractRector;
+use Rector\Renaming\Contract\MethodCallRenameInterface;
+use RectorPrefix202304\Webmozart\Assert\Assert;
```

Comment:
This `Assert` isn't available as a dependency. So we'd have to add it, or see if there is a non-namespaced namespace.

Reply:
Fixed